CHANGELOG.md

```diff
@@ -48,6 +48,20 @@ class DraggableItem extends React.Component {
 
 Note that the `draggableNode` is passed as a function because the ref `_listItem` is undefined until the component is mounted.
 
+## Service class accepts query parameters
+
+The `Service` class now accepts an object as it's second argument, deprecating the separate `onSuccess` and `onError` arguments.
+This allows you to pass in `onSuccess` and `onError` functions in the object, along with `params` if you need query parameters in your requests.
+
+### Examples
+
+Deprecated invocation:
+ - `service.get('1', onSuccessFunc, onErrorFunc)`
+
+New invocation:
+ - `service.get('1', { onSuccess: onSuccessFunc, onError: onErrorFunc })`
+ - `service.get('1', { onSuccess: onSuccessFunc, onError: onErrorFunc, params { key1: 'val1', key2: 'val2'} })`
```

Comment:
your example doesn't look right with the param value - shouldn't it be an object?

```
{ params: { key1: 'val1', key2: 'val2' } }
```

not 

```
params { key1: 'val1', key2: 'val2'}
```

Reply:
Just missing a colon

```
service.get('1', { onSuccess: onSuccessFunc, onError: onErrorFunc, params: { key1: 'val1', key2: 'val2' } })
```

Then the object is transformed by `axiosParams` to remove the callbacks.